package.json

```diff
@@ -91,9 +91,9 @@
     "devDependencies": {
         "electron": "^3.0.6",
         "electron-builder": "^20.29.0",
-        "electron-reload": "^1.2.5",
+        "electron-reload": "^1.5.0",
```

Comment:
Was there a reason this was updated?

Reply:
`npm audit` ran a bunch of critical package updates, so I ran `fix` on them. More a force of habit than anything